app/src/test/java/com/stevesoltys/seedvault/transport/backup/ApkBackupTest.kt

```diff
@@ -53,6 +54,12 @@ internal class ApkBackupTest : BackupTest() {
         mockkStatic(PackageUtils::class)
     }
 
+    @BeforeEach
+    fun setUp() {
+        every { settingsManager.backupApks() } returns true
+        every { settingsManager.isBackupEnabled(any()) } returns true
+    }
```

Comment:
For small things like these, I'd prefer to have them explicitly in each test, especially when the test outcome depends on it, e.g. `settingsManager.backupApks()`

Reply:
I would argue that establishing a default state for a "happy path" and then having tests which deviate from that path makes the tests easier to understand and reduces duplicate code.

This is subjective though so I've reverted the change.

Reply:
Thanks!